driver/copy_ami_driver.go

```diff
@@ -109,6 +109,25 @@ func (d *SDKCopyAmiDriver) Create(driverConfig resources.AmiDriverConfig) (resou
 	if err != nil {
 		d.logger.Printf("Error tagging AMI: %s, Error: %s ", *amiIDptr, err.Error())
 	}
+
+	for i := range driverConfig.SharedWithAccounts {
```

Comment:
Maybe use the `for _, value := range` form here since an index doesn't appear to be needed.

```golang
	for _, account := range driverConfig.SharedWithAccounts {
		_, err := ec2Client.ModifyImageAttribute(&ec2.ModifyImageAttributeInput{
```

Reply:
Ah, yes your right, that's better.

Done